crates/theme/src/settings.rs

```diff
@@ -157,20 +157,28 @@ impl ThemeSelection {
 
 #[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema)]
 pub struct ThemeSettingsContent {
+    /// The default font size for text in the UI
     #[serde(default)]
     pub ui_font_size: Option<f32>,
+    /// The name of a font to use for rendering text in the editor
```

Comment:
```suggestion
    /// The name of a font to use for rendering text in the UI.
```

Also, speaking of the 

> editor

part of the change, it is somewhat confusing: we have `buffer` in the settings names here and in the [code](https://github.com/zed-industries/zed/blob/main/crates/language/src/buffer.rs), whereas `editor` [means](https://github.com/zed-industries/zed/blob/main/crates/editor/src/editor.rs) something different in the code.

Maybe we can try phrasing docs like `/// The name of a font to use for rendering in text buffers.` in all related places with `editor`?

Reply:
agree. will check the strings for this.